config/gradle/utility.gradle

```diff
@@ -22,6 +22,15 @@ ext {
     templatesDir = 'templates'
 }
 
+// Prompts the user to enter a line of text
+def getUserString (String prompt) {
+    println ('\n' + (char)27 + '[1m*** ' + prompt + '\n') // Bold terminal text
```

Comment:
Looks nice in bash - does it also work in other shells, and in particular on Windows command line?
When in doubt, I suggest not doing the bold text trick.